spec/services/catalog/copy_portfolio_item_spec.rb

```diff
@@ -68,19 +68,42 @@
     end
 
     context "when the portfolio_item has service_plans" do
-      let(:params) { { :portfolio_item_id => portfolio_item.id, :portfolio_id => portfolio.id } }
+      context "when the service_plan base match topology" do
+        let(:params) { { :portfolio_item_id => portfolio_item.id, :portfolio_id => portfolio.id } }
+
+        before do
+          allow(Catalog::SurveyCompare).to receive(:changed?).and_return(false)
+
+          portfolio_item.service_plans.create!(
+            :base     => JSON.parse(File.read(Rails.root.join("spec", "support", "ddf", "valid_service_plan_ddf.json"))),
```

Comment:
Since we're overriding the return of `Catalog::SurveyCompare` do we need to do the `Json.parse` on real data here vs the `{}` on line 98, 99?

Reply:
Had to add another `allow` to make the other validation pass, but I was able to remove the `File.read` calls.